_posts/2018-03-15-this-week-in-rustfest-2.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "These weeks in RustFest 2: Diversity Tickets & CFP Last Call"
+authors:
+  - hoverbear
+---
+
+# CFP Last Call
+
+<div style="text-align: center">
+  <strong><a href="https://cfp.rustfest.eu/events/rustfest-paris">March 17th (tomorrow!) is the CFP deadline! No extensions!</a></strong>
+  <br>
+  <iframe src="https://giphy.com/embed/l3vRkWqpxQW3wm6ek" width="480" height="360" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
+</div>
+
+This is your very last chance to submit a presentation to RustFest. Due to our timeline this year we won't be doing an extension. Topic not quite refined yet? Submit it! Waffling between two ideas? Submit both! Scared to submit because you might have to speak? You can do it! Trust us!
+
+**PS** You can also submit it to [RustConf](https://cfp.rustconf.com/events/rustconf-2018)!
+
+
+# Diversity Sponsorships
+
+As with previous years we are again partnering with our lovely friends at [DiversityTickets.org](https://diversitytickets.org/) to support our community members which need assistance in attending RustFest. We welcome members of underrepresented groups to apply for this support.
```

Comment:
That deadline is **tomorrow**.
Also needs a direct link: https://diversitytickets.org/events/152